samples/bluetooth/fast_pair/locator_tag/include/app_fp_adv.h

```diff
@@ -32,11 +32,60 @@ enum app_fp_adv_mode {
 	APP_FP_ADV_MODE_NOT_DISCOVERABLE,
 };
 
-/** Set the Fast Pair advertising mode.
+/** Fast Pair advertising trigger. */
+struct app_fp_adv_trigger {
+	/* Identifier. */
+	const char *id;
+
+	/* True if the trigger is enabled, false otherwise.
+	 * The API user must not update the value of this field and
+	 * should use the @ref app_fp_adv_request function for this purpose.
+	 */
+	bool enabled;
+};
+
+/** Register a trigger for Fast Pair advertising.
+ *
+ *  @param _name Trigger structure name.
+ *  @param _id Trigger identifier.
+ */
+#define APP_FP_ADV_TRIGGER_REGISTER(_name, _id)				\
+	BUILD_ASSERT(_id != NULL);					\
+	static STRUCT_SECTION_ITERABLE(app_fp_adv_trigger, _name) = {	\
+		.id = _id,						\
+	}
+
```

Comment:
```suggestion
/** Fast Pair advertising trigger. */
/** Opaque type representing an advertising trigger */
struct app_fp_adv_trigger;

/** Register a trigger for Fast Pair advertising.
 *
 *  @param name Trigger name.
 *  @param trigger Trigger instance.
 *
 *  @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
 */
int app_fp_adv_trigger_register(const char *trigger_name, struct app_fp_adv_trigger **trigger)

```

Reply:
Why should we register subscribers dynamically? It seems not to simplify anything (see my comment above for details).

Reply:
Let's discuss it.